### ISSUE TITLE
Build Error after adding the Plugin

### ISSUE THREAD

**Issue by baltejsingh88** [Wizcorp/phonegap-facebook-plugin#1318]
Hi

I added this plugin into my project and while building for iOS i got this error.

![image](https://cloud.githubusercontent.com/assets/16228380/19226393/469bf570-8ec8-11e6-9ed8-9fe75b6bc302.png)
## System Information:

Cordova CLI: 6.1.1
Gulp version:  CLI version 3.8.11
Gulp local:   Local version 3.9.0
Ionic Framework Version: 1.3.1
Ionic CLI Version: 2.1.0
Ionic App Lib Version: 2.0.0-beta.20
ios-deploy version: 1.8.6 
ios-sim version: 3.1.1 
OS: Mac OS X El Capitan
Node Version: v5.0.0
Xcode version: Xcode 8.0 Build version 8A218a

**Comment by markfarl** [comment]
Me too, on xcode it says 

> /Plugins/phonegap-facebook-plugin/FacebookConnectPlugin.m:27:44: No visible @interface for 'CDVPlugin' declares the selector 'initWithWebView:'

**Comment by baltejsingh88** [comment]
@markfarl exactly...

**Comment by markfarl** [comment]
Its issues with cordova version I think, you need to use this one here boss

https://github.com/jeduan/cordova-plugin-facebook4

works for me now, although it builds, if you use OSX 10 you have to do stuff with the keychain, those issues are covered in the issues in that repo though

**Comment by baltejsingh88** [comment]
@markfarl  yes i am using that already now.. but it doesnot open the Facebook app, it always open the Modal to login to FB.. can you tell me how to handle that so that it used the native facebook app?

**Comment by markfarl** [comment]
Ah, I hadnt tested it with fb app yet, crap I assumed it did. 

I'll install fb app to see if it does on mine now thanks
